R/data_inventory_table.R

```diff
@@ -467,3 +467,128 @@ pt_data_inventory_notes <- function(bq = c("BQL", "BLQ"), drop_bql = FALSE, note
   )
   ans
 }
+
+
+#' Data inventory by covariate
+#'
+#' Generates a table showing numbers of subjects and observations, stratifying
+#' by multiple categorical covariates.
+#'
+#' @inheritParams pt_data_inventory
+#'
+#' @param cols data columns containing discrete data items for grouped data
+#' inventory summaries.
+#' @param level_width width of the `level` column, specified in `cm`.
```

Comment:
Based on this docstring alone (and not reading the code), I don't think it'd be clear to me the "level" column is referring to.